utopia-remix/app/util/common.ts

```diff
@@ -0,0 +1,6 @@
+export function asNumber(value?: unknown): number {
+  if (!['string', 'number'].includes(typeof value) || value === '') {
```

Comment:
shouldn't this be a simpler try/catch `parseInt` if it's a string, and just the number otherwise?